internal/commands/reminder.go

```diff
@@ -99,30 +127,36 @@ func requestStatus(ctx context.Context, client bot.Client, logger log.Logger, re
 		if err != nil {
 			logger.Error(
 				ctx,
-				"command/reminder.requestStatus convertTimestamp error",
+				log.Trace(),
+				log.NewValue("action", "convertTimestamp"),
 				log.NewValue("channelID", incident.ChannelId),
+				log.NewValue("channelName", incident.ChannelName),
 				log.NewValue("error", err),
 			)
 			return
 		}
 
-		if timeMessage.Before(time.Now().Add(-setRecurrence(incident))) {
-			sendNotification(ctx, logger, client, incident)
-			return
-		} else {
+		if timeMessage.After(time.Now().Add(-setRecurrence(incident))) {
```

Comment:
Why have you inverted this logic?

Reply:
I reversed the order to make more sense. There was an unnecessary `else`, where it checked the time of the pinned message to send an alert, otherwise it generated an ok message log. Then, removing `else`, the log message is sent without context.

In addition, it also follows the same principle as other checks, such as SLA verification, canStopReminder and snoozedUntil.

Reply:
Perfect makes total sense!